### DISCUSSION TITLE
plss remove portguese translation from start page

### DISCUSSION THREAD

**Discussion by luan661** [axelboc/anki-ultimate-geography#550] [General]
remove "portuguese" deck from the main page description since it doesn't exist in download page (v5.0)
![2022-07-07 22_20_56-Window](https://user-images.githubusercontent.com/60822290/177897798-18919abe-3c44-41ce-bc23-649e06379f26.png)
![2022-07-07 22_23_39-Window](https://user-images.githubusercontent.com/60822290/177897900-4ff606ce-27ad-4321-baf4-a60d57f226bb.png)

**Comment by aplaice** [comment]
Sorry for the confusion!

The Portuguese version was translated after the release of version 5.0, so it's not _yet_ available on the download page.

Hopefully, we'll be releasing 5.1 soon-ish, so Portuguese will also be easily available.

**Reply by luan661** [reply]
we hope so
thanks for reply!🎈🎈🎈
